pokemongo_bot/api_wrapper.py

```diff
@@ -8,13 +8,18 @@
 from pgoapi.pgoapi import PGoApi, PGoApiRequest, RpcApi
 from pgoapi.protos.POGOProtos.Networking.Requests_pb2 import RequestType
 
-from human_behaviour import sleep
+from human_behaviour import sleep, gps_noise_rng
 
 class ApiWrapper(PGoApi):
-    def __init__(self):
+    def __init__(self, replicate_gps_noise=False, gps_noise_range=0.00025):
         PGoApi.__init__(self)
         self.useVanillaRequest = False
 
+        if replicate_gps_noise:
+            self.gps_noise_range = gps_noise_range
```

Comment:
Some sources say gps altitude precision is worse than lat, and lng to make those two precision better.
Since I'm not sure if the app is actually sending altitude, and the alt should be handled differently,
I will separate altitude conf.